File: python/Iniciante/URI1043.py
linha = input().split()
a, b, c = list(map(float, linha))

if ((a < b + c) and (b < a + c) and (c < b + a)):
  perimetro = a + b + c;
  print(f"Perimetro = {perimetro:.1f}");
else:
  area = (a + b) * c / 2
  print(f"Area = {area:.1f}")

File: java/iniciante/URI1043.java
package URI;

import java.util.Scanner;

public class URI1043 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double A = scanner.nextDouble();
        double B = scanner.nextDouble();
        double C = scanner.nextDouble();

        if ((A < B + C) && (B < A + C) && (C < B + A)) {
            double perimetro = A + B + C;
            System.out.printf("Perimetro = %.1f\n", perimetro);
        } else {
            double area = (A + B) * C / 2;
            System.out.printf("Area = %.1f\n", area);
        }
    }
}
